elasticsearch/v7/elastic_v7.go

```diff
@@ -0,0 +1,45 @@
+package v7
+
+import (
+	"strings"
+
+	"github.com/HomesNZ/env"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	v4 "github.com/aws/aws-sdk-go/aws/signer/v4"
+	"github.com/sha1sum/aws_signing_client"
+	"github.com/sirupsen/logrus"
+
+	elastic "github.com/olivere/elastic/v7"
+)
+
+// Gets a connection to AWS OpenSearch - depends on ENV
+//
+//	ELASTICSEARCH_URLS
+//	+ AWS_ACCESS_KEY_ID or AWS_ACCESS_KEY
+//	+ AWS_SECRET_ACCESS_KEY or AWS_SECRET_KEY
+func Conn() *elastic.Client {
+	log := logrus.WithField("package", "elasticsearch")
+	signer := v4.NewSigner(credentials.NewEnvCredentials())
+	awsClient, err := aws_signing_client.New(signer, nil, "es", "ap-southeast-2")
+	if err != nil {
+		log.Errorln("error creating aws signer")
+		panic(err)
+	}
+
+	urls := strings.Split(env.MustGetString("ELASTICSEARCH_URLS"), ";")
```

Comment:
I would prefer us to use env.GetString, and if it's empty -return an error.

Reply:
makes sense, and is a good time to change the signature